webpack/reducers/forms_reducer.js

```diff
@@ -1,13 +1,20 @@
 import {
+  FETCH_FORM_FULFILLED,
   FETCH_FORMS_FULFILLED
 } from '../actions/types';
 import _ from 'lodash';
 
-const initialState = {'loading': true};
+const initialState = null;
```

Comment:
That makes sense, I removed 'loading' as a field for that reason, it wasn't always going to be there.

Anyway, I brought mine more in line with yours.